opentelemetry-sdk/src/metrics/mod.rs

```diff
@@ -1251,6 +1251,118 @@ mod tests {
         counter_multithreaded_aggregation_helper(Temporality::Cumulative);
     }
 
+    #[tokio::test(flavor= "multi_thread", worker_threads = 1)]
+    async fn synchronous_instruments_cumulative_with_gap_in_measurements() {
+        // Run this test with stdout enabled to see output.
+        // cargo test synchronous_instruments_cumulative_with_gap_in_measurements --features=testing -- --nocapture
+
+        synchronous_instruments_cumulative_with_gap_in_measurements_helper("counter");
+        synchronous_instruments_cumulative_with_gap_in_measurements_helper("updown_counter");
+        synchronous_instruments_cumulative_with_gap_in_measurements_helper("histogram");
+
+        /* Synchronous Gauge has an aggregation bug. Uncomment the code below to run the test for gauge
+        once this issue is fixed: https://github.com/open-telemetry/opentelemetry-rust/issues/1975
+        */
+
+        // synchronous_instruments_cumulative_with_gap_in_measurements_helper("gauge");
```

Comment:
This test fails for synchronous Gauge. We need to fix #1975 before we can run the test for Gauge.